src/test/java/org/candlepin/dto/api/v1/GuestIdArrayElementTranslatorTest.java

```diff
@@ -75,9 +75,9 @@ protected void verifyOutput(GuestId source, GuestIdDTOArrayElement dto, boolean
             assertEquals(source.getId(), dto.getId());
             assertEquals(source.getGuestId(), dto.getGuestId());
             assertEquals(source.getCreated(), dto.getCreated() != null ?
-                new Date(dto.getCreated().toInstant().toEpochMilli()) : null);
+                (new Date(dto.getCreated().toInstant().toEpochMilli())) : null);
```

Comment:
We can just use .getTime() to get the milliseconds straight from the date object

Reply:
created is an OffsetDateTime. Not sure I can reduce what is here.

Reply:
Ahh, my mistake; didn't process that this was from the DTO. Also didn't notice that we're getting millis to convert to a date. In that case, we can skip the millis bit entirely:

```
Date.from(dto.getCreated().toInstant())
```

It's too bad there's no shared interface between Instant and OffsetDateTime. The amount of `.toInstant()`s we're going to have around the code is going to be high for the foreseeable future.